packages/core/src/services/workspaces/update.ts

```diff
@@ -1,23 +1,22 @@
 import { eq } from 'drizzle-orm'
 
 import type { Workspace } from '../../browser'
+import { database } from '../../client'
 import { Result, Transaction } from '../../lib'
 import { workspaces } from '../../schema'
 
-export async function updateWorkspace({
-  workspace,
-  name,
-}: {
-  workspace: Workspace
-  name: string
-}) {
-  return await Transaction.call(async (tx) => {
+export async function updateWorkspace(
+  workspace: Workspace,
+  values: Partial<Workspace>,
+  db = database,
+) {
+  return Transaction.call<Workspace>(async (tx) => {
```

Comment:
Make sure what `values` we're allowing to pass from the user. We shouldn't allow to change values other than name and the new default fields. Although it's okay if that restriction is somewhere else I guess

Reply:
Okay, will do, I based the changed on the `user update` service `packages/core/src/services/users/update.ts`, maybe in another PR we should restrict the fields too.